src/Infrastructure/Util/include/ESMC_Util.h

```diff
@@ -58,6 +58,11 @@ enum ESMC_Decomp_Flag {ESMC_DECOMP_INVALID=0,
                        ESMC_DECOMP_BALANCED, ESMC_DECOMP_RESTFIRST,
                        ESMC_DECOMP_RESTLAST, ESMC_DECOMP_CYCLIC};
 
+// end flag
+typedef enum ESMC_End_Flag { ESMF_END_NORMAL=1,
+                             ESMF_END_KEEPMPI,
```

Comment:
Shouldn't these be ESMC_... (e.g. ESMC_END_NORMAL)?

Reply:
Ah, yeah. I originally placed `ESMC_End_Flag` in `ESMCI_Util.h`, where those enum values are prefixed with `ESMF_*`, but didn't notice the naming difference.

Reply:
Fixed in [436de54](https://github.com/esmf-org/esmf/pull/234/commits/436de5445b5f9ffede2bec1361c0776f1ac688aa)